robot_calibration/src/calibrate.cpp

```diff
@@ -292,7 +292,7 @@ int main(int argc, char** argv)
       opt.optimize(params, data, verbose);
       if (verbose)
       {
-        std::cout << "Parameter Offsets:" << std::endl;
+        std::cout << "Parameter Offsets with cal_steps:" << std::endl;
         std::cout << opt.getOffsets()->getOffsetYAML() << std::endl;
```

Comment:
Should you instead use ros logging statements here instead of court?   Instead of the verbose bool you could just always log these things to the DEBUG level and then a user can turn them on/off based on that?